src/main/java/org/spongepowered/common/mixin/core/entity/projectile/MixinEntityWitherSkull.java

```diff
@@ -80,4 +94,25 @@ public void writeToNbt(NBTTagCompound compound) {
         }
     }
 
+    @Nullable
+    @Redirect(method = "onImpact(Lnet/minecraft/util/MovingObjectPosition;)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/World;"
```

Comment:
For readability, just set the targets to private static final fields since they're inlined in the compiler anyways.